dropwizard-db/src/main/java/io/dropwizard/db/DataSourceFactory.java

```diff
@@ -403,19 +403,19 @@ public int get() {
     private boolean autoCommentsEnabled = true;
 
     @NotNull
-    @MinDuration(1)
+    @MinDuration(value = 0, unit = TimeUnit.MILLISECONDS, inclusive = false)
     private Duration evictionInterval = Duration.seconds(5);
 
     @NotNull
-    @MinDuration(value = 50, unit = TimeUnit.MILLISECONDS)
+    @MinDuration(value = 0, unit = TimeUnit.MILLISECONDS, inclusive = false)
```

Comment:
I don't think we want to let this one be that low. #2051 reduced the minimum from one second to 50ms and I don't see a reason to deviate from this.